lib/parser.js

```diff
@@ -190,13 +190,27 @@ async function validateAndConvertMessage(msg, originalAsyncAPIDocument, fileForm
 
   const schemaFormat = msg.schemaFormat || DEFAULT_SCHEMA_FORMAT;
 
-  await PARSERS[String(schemaFormat)]({
+  //set for the test
+  global.tempPathToPayload = pathToPayload;
```

Comment:
we should not mix code related to test with main library logic. Why do you have it, most of the error message is anyway hardcoded in the test, then this part can be too